### PR TITLE
Add the security guidelines to the readme

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by renetapopova** [comment]
Do you think we'll benefit if we add these references to the guidelines:

* https://cheatsheetseries.owasp.org/cheatsheets/Logging_Vocabulary_Cheat_Sheet.html
* https://cheatsheetseries.owasp.org/cheatsheets/Logging_Cheat_Sheet.html

File: README.adoc [hunk 1]
```diff
@@ -64,6 +64,39 @@ It could be omitted as the title already says that it is an experimental feature
...
+For login errors, password validations, or any security-related processes, use generic error messages.
+For example, use "Login failed" instead of specifying whether the username or password was incorrect.
+
+* Do not expose internal information.
+Never reveal information about the internal state of the application, like file paths, database information, or stack traces, as these can provide attackers with clues about potential vulnerabilities.
+Stack traces are safe to have in debug.log but nowhere else.
+
+* Do not use error messages from underlying libraries/dependencies (e.g. Java runtime errors) but always catch the error and throw your own error message.
+
+* For wrapped errors, it is safe to reuse error messages only if they are coming from Neo4j, i.e. have a status.
```
Comment (Lojjs) [inline README.adoc:88]:
```suggestion
* Do not use error messages from underlying libraries/dependencies (e.g. Java runtime errors) but always catch the error and throw your own error message. It is safe to reuse error messages only if they are coming from Neo4j, i.e. have a Neo4j status code.
```

**Review by Lojjs — CHANGES_REQUESTED** [review]
Really good! Just one thing which I think can be adjusted to be clearer, see suggestion

**Comment by Lojjs** [comment]
> Do you think we'll benefit if we add these references to the guidelines:
> 
> * https://cheatsheetseries.owasp.org/cheatsheets/Logging_Vocabulary_Cheat_Sheet.html
> * https://cheatsheetseries.owasp.org/cheatsheets/Logging_Cheat_Sheet.html

Maybe if we add them in the bottom of the section, with a note that clarifies that it is further information for those interested and not a requirement to read?

**Review by Lojjs — APPROVED** [review]

**Pull request merged**